webapp/src/ts/services/enketo.service.ts

```diff
@@ -392,14 +410,14 @@ export class EnketoService {
     });
   }
 
-  private renderForm(selector, formDoc, instanceData, editedListener, valuechangeListener) {
+  private renderForm({ selector, formDoc, instanceData, editedListener, valuechangeListener, titleKey = undefined }) {
```

Comment:
These parameters are getting long, can you please name the object:  `private renderForm(myParams)` then use values from there, it helps explaining the code too. `myParams.selector`

Reply:
Same question with renderContactForm(...)